index.html

```diff
@@ -1612,45 +1633,49 @@ <h3 id="applying">
           <code>dir</code> member
         </h3>
         <p>
-          The <dfn><code>dir</code> member</dfn> specifies the <dfn>base
+          The <code><dfn>dir</dfn></code> member specifies the <dfn>base
           direction</dfn> for the <a>directionality-capable members</a> of the
           <a>manifest</a>. The <code>dir</code> member's value can be set to
           one of the <a>text-direction values</a>.
         </p>
+        <pre class="idl">
+            [NoInterfaceObject]
+            enum TextDirectionType { "ltr", "rtl", "auto" };
+        </pre>
         <p>
           The <dfn>directionality-capable members</dfn> are:
         </p>
         <ul>
           <li>
-            <a><code>description</code> member</a>.
+            <a><code>description</code></a> member.
```

Comment:
s/`<a><code>description</code></a> member.`/`<a>description</a> member.`

Same for the rest. You should never need `<code>` anywhere.